### PR TITLE
Add global typeography constants

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Added typeography types
- Added typeography constants
- Exported typeography types and constants

## Issues
Updated #22231

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `73.754 kB`<br />`22.494 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `45.196 kB`<br />`13.126 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `20.965 kB`<br />`6.605 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `21.918 kB`<br />`6.919 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `22.005 kB`<br />`6.582 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `32.408 kB`<br />`8.986 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `38.635 kB`<br />`10.146 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `34.175 kB`<br />`9.567 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `41.431 kB`<br />`10.865 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |    `43.31 kB`<br />`9.921 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `57.884 kB`<br />`16.524 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `53.169 kB`<br />`15.238 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `7.756 kB`<br />`3.299 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `9.321 kB`<br />`3.816 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.728 kB`<br />`3.323 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>                                                                              |  `60.564 kB`<br />`20.751 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `179.206 kB`<br />`50.033 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.671 kB`<br />`10.696 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |    `15.455 kB`<br />`5.57 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `10.179 kB`<br />`3.995 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `21.775 kB`<br />`7.228 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `8.441 kB`<br />`3.541 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.176 kB`<br />`4.545 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `110.192 kB`<br />`33.528 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `113.367 kB`<br />`33.991 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `100.693 kB`<br />`30.67 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |     `6.272 kB`<br />`2.17 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                                                         |    `23.21 kB`<br />`8.084 kB` |
| <samp>react-priority-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                      |   `10.792 kB`<br />`4.125 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `14.079 kB`<br />`5.274 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `16.616 kB`<br />`6.281 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `25.619 kB`<br />`8.287 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `42.003 kB`<br />`11.886 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `17.224 kB`<br />`5.856 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `24.333 kB`<br />`8.019 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `10.867 kB`<br />`4.269 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `14.183 kB`<br />`4.61 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |    `20.672 kB`<br />`7.07 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |    `29.676 kB`<br />`6.61 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |     `18.492 kB`<br />`5.3 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `42.907 kB`<br />`14.761 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/15e9cd77b663bfc839f05b9f5a580bf6556faa59'>15e9cd77b663bfc839f05b9f5a580bf6556faa59</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/Avatar-mount_base.html">945</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/Avatar-mount.html">900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/Button-mount_base.html">581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/Button-mount.html">577</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProvider-mount_base.html">1950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProvider-mount.html">1907</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-mount.html">322</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">236</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">318</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/MakeStyles-mount_base.html">1559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22610/perf-test-react-components/MakeStyles-mount.html">1575</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-theme/src/global/typeographyStyles.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { tokens } from '../tokens';
+import { TypeographyStyles } from '../types';
+
+/**
+ * Global typeography styles (fontSize, fontWeight, and lineHeight)
+ */
+export const typeographyStyles: TypeographyStyles = {
```
Comment (miroslavstastny) [inline packages/react-theme/src/global/typeographyStyles.ts:7]: `Text` already contains typography styles (which are not up to date): https://github.com/microsoft/fluentui/blob/5a6f0d55d87f70d67dd1567ce97e637e51b07863/packages/react-text/src/typographyStyles/typographyStyles.ts#L7.

We should remove one of them and keep just a single instance.

**Review by miroslavstastny — CHANGES_REQUESTED** [review]
Previously we agreed that instead of using the typography directly, we will reuse `Text` component. If we now feel we should do it differently, let's solve that first.
cc @microsoft/cxe-prg

File: packages/react-theme/src/global/typeographyStyles.ts (hunk 1 above)
Comment (GeoffCoxMSFT) [reply packages/react-theme/src/global/typeographyStyles.ts:7]: I will do that update in a follow up PR immediately after this one is in.  Does that work?

**Review by miroslavstastny — APPROVED** [review]
LGTM, Added @microsoft/cxe-prg as a reviewer.

**Pull request merged**